L6_MaxProductOfThree.py

```diff
@@ -78,11 +78,15 @@ def solution(A):
     :param A: list[int]
     :return: [int]
     """
-    A.sort()
-    return max(
-        A[0] * A[1] * A[-1],
-        A[-3] * A[-2] * A[-1]
-    )
+    max=0
+    
+    a=sorted(a)
+    a.reverse()
+
+    max=(a[0] * a[1] * a[2])
+    
+    return max
+    
```

Comment:
This solution is incorrect, and scores 44%.
https://app.codility.com/demo/results/training3579PV-X5Y/

[As addressed in the notes:](https://github.com/johnmee/codility/blob/master/L6_MaxProductOfThree.py#L54)

> Sorting the array then multiplying the three biggest numbers will only get you a score of 44%.
The 'trick' is that two negatives make a positive.
eg: [-100, -100, -100, 0, 50, 50, 50]
The product of the top 3 is 125000,
but the product of the bottom two (-100, 100) with the top one (50) is 500000!